src/test/java/scm/address/logic/parser/AddressBookParserTest.java

```diff
@@ -258,6 +257,14 @@ public void parseCommand_listScheduleCommand() throws Exception {
         assertTrue(parser.parseCommand(ListScheduleCommand.COMMAND_WORD + " 5") instanceof ListScheduleCommand);
     }
 
+    @Test
+    public void parseCommand_listOngoingScheduleCommand() throws Exception {
```

Comment:
Consider changing function name to (command)_(input)_(expectation i.e:success/fail/exception) form? so it'd be parseCommand_listOngoingScheduleCommand_success() ?

Reply:
I think the current name is good enough, what do you think? It is consistent with the other function names in the class.